doc/changes/unreleased.md

```diff
@@ -10,3 +10,9 @@ and replaces them with `format:fix` and `format:check`.
 ## Feature
 
 * #614: Replaced `path_filters` with `BaseConfig.add_to_excluded_python_paths` and `BaseConfig.excluded_python_paths`
+* #621: Moved path specifications into `BaseConfig`
```

Comment:
Is this a breaking change from the perspective of the projects using the PTB?

Reply:
Maybe add to the summary for the next release, that projects when updating the PTB, should also take care about potential custom nox files, potentially using `root`, etc.